src/modules/units/components/UnitGroupCard.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Paper, Stack, Typography } from '@mui/material';
+import CommonMenuButton, {
+  CommonMenuItem,
+} from '@/components/elements/CommonMenuButton';
+import UnitUtilizationByUnitType from '@/modules/units/components/UnitUtilizationByUnitType';
+import { UnitGroupCapacityFieldsFragment } from '@/types/gqlTypes';
+
+interface Props {
+  unitGroup: UnitGroupCapacityFieldsFragment;
+  menuItems?: CommonMenuItem[];
+}
+
+const UnitGroupCard: React.FC<Props> = ({ unitGroup, menuItems }) => {
+  return (
+    <Paper sx={{ px: 2, pt: 1, pb: 2 }}>
+      <Stack
+        justifyContent='space-between'
+        direction='row'
+        alignItems='flex-start'
+      >
+        <div>
+          <Typography variant='overline'>Unit Group</Typography>
+          <Typography variant='h5' sx={{ mb: 1 }}>
```

Comment:
```suggestion
          <Typography component='h2' variant='h5' sx={{ mb: 1 }}>
```

Reply:
Also, I'm not totally sure about all merits/disadvantages of this approach, but we could put the overline text _in_ the heading text. That way it would get read by screen readers as part of this heading, as opposed to part of the previous section's content. We do this in `PageTitle.tsx`:
https://github.com/greenriver/hmis-frontend/blob/b325723d685dac1216bee699f17547703401daae/src/components/layout/PageTitle.tsx#L32-L40